### ISSUE TITLE
Support sub-languages

### ISSUE THREAD

**Issue by niccokunzmann** [niccokunzmann/mundraub-android#212]
Currently, the app only supports languages like "de" or "en". There are sub-languages like "fr-FR" and "de-AT". These should be supported, too.
If we introduce them now, we will get errors. https://github.com/niccokunzmann/mundraub-android/issues/130

**Comment by Flexmaen** [comment]
I guess this is related to the Transifex request I did. I did translate the strings.xml, but as far as I understand this won't help much now.

**Comment by niccokunzmann** [comment]
Yes, I saw your request on Transifex. There is a German version already, so I hope you did not put too much time into de-AT. Did you translate something to de-AT?

**Comment by Flexmaen** [comment]
Especially fruits sometimes have completely different names in Germany and Austria, so a translation even for the website would be fine, since some words are unknown or can be misunderstood.

However, translating didn't take too long, it just concerns some fruit names.

**Comment by niccokunzmann** [comment]
Okay. I understand. Also, I know that in other languages than de, the names and strings are sometimes very different, too. I will do some research as just adding a sub-language will break the build.

**Comment by niccokunzmann** [comment]
- https://stackoverflow.com/a/30028371
- https://developer.android.com/training/basics/supporting-devices/languages
- found the hint: `res/values-en-rGB/strings.xml` in https://developer.android.com/guide/topics/resources/localization

**Comment by niccokunzmann** [comment]
To fix this issue,
- [ ] we need to transform the files from `app/src/main/res/values-fr_FR` format to `res/values-en-rGB/strings.xml` format.
- [ ] document how exactly to translate the app: first the common language (de), then the country (de-DE) only with differences.

**Comment by niccokunzmann** [comment]
A new build will bring clarity if it works. https://travis-ci.org/niccokunzmann/mundraub-android/builds/445283265?utm_source=github_status&utm_medium=notification
